src/middleware/HTTPMessageHandler.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @module HTTPMessageHandler
+ */
+
+import { Context } from "../IContext";
+import { Middleware } from "../IMiddleware";
+
+/**
+ * @class
+ * Class for HTTPMessageHandler
+ * @extends Middleware
+ */
+export class HTTPMessageHandler implements Middleware {
+
+
+    /**
+     * @public
+     * @async
+     * To execute the current middleware
+     * @param {Context} context - The request context object
+     * @returns A promise that resolves to Nothing
+     */
+    public async execute(context: Context): Promise<void> {
+        try {
+            context.response = await fetch(context.request, context.options);
+            return;
```

Comment:
return statement not required.

Reply:
Its just to bring in the readability that it doesn't return anything.